pages/common/export.md

```diff
@@ -0,0 +1,15 @@
+# export
+
+> Command to set environment variables.
```

Comment:
Can't the export command be used to 'export' a variable's value to sub-processes too?

Reply:
It's open to interpretation, but yes. `export` defines variables in the **current** environment and therefore sub-processes can access them as well.
Shall I point out in the description that variables that are set with `export` are only present in the current session and will not be persisted?